packages/config/src/config.js

```diff
@@ -1,35 +1,36 @@
 /* eslint-disable camelcase */
 const path = require('path');
-const proxy = require('@redhat-cloud-services/frontend-components-config-utilities/proxy');
+const proxy = require('@redhat-cloud-services/frontend-components-config-utilities/standalone/proxy');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
-let rewriteLineCounter = 0;
-
 module.exports = ({
-    port,
+    port = 1337,
     publicPath,
     appEntry,
     rootFolder,
     https,
     mode,
     appName,
     useFileHash = true,
-    betaEnv = 'ci',
+    betaEnv,
```

Comment:
This isn't enough to determine which build repos to checkout, so I've deprecated it in favor of `env`.